components/linear_app/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/linear_app",
-  "version": "0.5.5",
+  "version": "0.5.6",
```

```suggestion
  "version": "0.6.0",
```
We should update to a new minor version since we're adding new features